flake.nix

```diff
@@ -18,6 +18,7 @@
   inputs.sops-nix.inputs.nixpkgs-stable.follows = "nixpkgs-stable";
   inputs.sops-nix.inputs.nixpkgs.follows = "nixpkgs";
   inputs.sops-nix.url = "github:Mic92/sops-nix";
+  inputs.buildbot-nix.url = "github:Mic92/buildbot-nix";
```

Comment:
This will cause changes in `flake.lock`, which is expected. However, the [buildbot-nix `flake.nix`](https://github.com/Mic92/buildbot-nix/blob/main/flake.nix) defines inputs that we also define in our `flake.nix`:
 - `nixpkgs`
 - `treefmt-nix`

In order to avoid duplication of these inputs, please have them follow ours.

Reply:
This should be resolved by https://github.com/ngi-nix/ngipkgs/pull/228/commits/f0536850d82beb3fa029393fcbf66cda4cde555c. I don't see treefmt-nix being used anymore, I believe it was removed in 6060fee and removed from the lockfile in ca7272ef267b8e9d3bda032fcd952c50e13866bf.